sesman/sesman.ini.in

```diff
@@ -64,10 +64,19 @@ IdleTimeLimit=0
 Policy=Default
 
 [Logging]
+; Note: Log levels can be any of: core, error, warning, info, debug, or trace
 LogFile=xrdp-sesman.log
-LogLevel=DEBUG
-EnableSyslog=1
-SyslogLevel=DEBUG
+LogLevel=INFO
+EnableSyslog=true
+#SyslogLevel=INFO
+#EnableConsole=false
+#ConsoleLevel=INFO
+#EnableProcessId=false
+
+[Logging_PerLogger]
```

Comment:
I don't like `_` (underscore) here. `SessionVariables` is spelled camel case.

Reply:
Fixed in commit 04dff6a to use camel case for the config names.